lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -579,6 +624,47 @@ def send_community=(val)
       config_set('bgp_neighbor_af', 'send_community', @set_args)
     end
 
+    def send_comm_iosxr_set(val)
+      if val[/none/]
```

Comment:
Original code which I copied - but it does seem unnecessary. I'll check with Chris.